cmake/platform/windows.cmake

```diff
@@ -130,20 +130,23 @@ function(add_python_executable)
   endif()
 
   if(WIN32)
+    get_filename_component(WRAPPER_NAME
+      "${ARG_TARGET_NAME}"
```

Comment:
The target name is different from the script name. And the latter is what needs to be used at least for the `OUTPUT_NAME` below.

Reply:
@dirk-thomas Sorry I don't follow here. Can you elaborate more what change I should make here?

Reply:
The name of the installed script is passed to the function as `SCRIPT_NAME`. Changing it to use `TARGET_NAME` instead is wrong. E.g.:

`add_python_executable(SCRIPT_NAME "name_the_user_sees.py" TARGET_NAME cmake_target_name_a_user_does_not_care_about DESTINATION ...)`

Reply:
@dirk-thomas Ahh... I see what's the problem there. Now it is fixed.